### PR TITLE
feat: add Transcoder samples

### PR DESCRIPTION
**Pull request by irataxy**
b: 160888570

### CODE REVIEW

**Review by ipotuzhnov — APPROVED** [review]

**Comment by fhinkel** [comment]
Thanks for the PR! But I think the sample should go here: https://github.com/googleapis/nodejs-video-transcoder/tree/master/samples

File: media/transcoder/system-test/transcoder.test.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+it('should create a job from preset', () => {
+  let output = execSync(
+    `node createJobFromPreset.js ${projectId} ${location} ${inputUri} ${outputUriForPreset} ${preset}`,
+    {cwd}
+  );
+  assert.ok(
+    output.includes(`projects/${projectNumber}/locations/${location}/jobs/`)
+  );
+  let id = output.toString().split('/');
```
[the remark this answers is not in the document]
Comment (irataxy) [reply media/transcoder/system-test/transcoder.test.js:121]: Done in https://github.com/googleapis/nodejs-video-transcoder/pull/8

File: media/transcoder/system-test/transcoder.test.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+it('should create a job template', () => {
+  const output = execSync(
+    `node createJobTemplate.js ${projectId} ${location} ${templateId}`,
+    {cwd}
+  );
+  assert.ok(output.includes(templateName));
+});
+
+it('should get a job template', () => {
```
[the remark this answers is not in the document]
Comment (irataxy) [reply media/transcoder/system-test/transcoder.test.js:90]: Refactored in https://github.com/googleapis/nodejs-video-transcoder/pull/8

File: media/transcoder/system-test/transcoder.test.js [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+const outputUriForTemplate = `gs://${bucketName}/test-output-template/`;
+const outputUriForAdHoc = `gs://${bucketName}/test-output-adhoc/`;
+
+const cwdDatasets = path.join(__dirname, '../../testdata');
+const cwd = path.join(__dirname, '..');
+
+const resourceFile = `../testdata/${testFileName}`;
+const installDeps = 'npm install';
+
+// Run npm install on datasets directory because modalities
```
[the remark this answers is not in the document]
Comment (irataxy) [reply media/transcoder/system-test/transcoder.test.js:44]: Refactored in https://github.com/googleapis/nodejs-video-transcoder/pull/8

File: media/transcoder/deleteJob.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  async function deleteJob() {
+    // Construct request
+    const request = {
+      name: transcoderServiceClient.jobPath(projectId, location, jobId),
+    };
+    const [response] = await transcoderServiceClient.deleteJob(request);
+    console.log('Deleted job');
+  }
+
+  deleteJob();
```
[the remark this answers is not in the document]
Comment (irataxy) [reply media/transcoder/deleteJob.js:42]: https://github.com/googleapis/nodejs-video-transcoder/pull/8

File: media/transcoder/system-test/transcoder.test.js [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+// require bootstrapping datasets, and Kokoro needs to know
+// to install dependencies from the datasets directory.
+assert.ok(execSync(installDeps, {cwd: `${cwd}/../transcoder`, shell: true}));
+before(async () => {
```
[the remark this answers is not in the document]
Comment (irataxy) [reply media/transcoder/system-test/transcoder.test.js:48]: https://github.com/googleapis/nodejs-video-transcoder/pull/8

**Pull request closed without merging**